pages/web_cloud/domains/how_to_connect_domain_to_google_site/guide.fr-fr.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Comment connecter un nom de domaine OVHcloud à un Google Site"
 excerpt: "Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour le connecter à un Google Site"
-updated: 2024-09-26
+updated: 2024-10-03
```

Comment:
Why changing the date? 
If no new information is added, the date should be kept as 2024-09-26 (same thing for all translations)